### PR TITLE
implement SwiperJS

### PR DESCRIPTION
**Pull request by Elgin-Schultz**
Please note, as data is not yet fully implemented only first entry works!

### CODE REVIEW

File: components/GlobalStyles.js [hunk 1]
```diff
@@ -51,15 +55,89 @@ mark { background-color: white;
...
+    width: 1rem;
 }
 
+.swiper-pagination-vertical {
+    
+    
+    z-index: 99999
+    }
+    
+
```
Comment (julialoeschel) [inline components/GlobalStyles.js:140]: It looks like prettier is not working, your blank lines are all over the place ;)

File: next.config.js [hunk 2]
```diff
@@ -15,4 +15,7 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+module.exports = {
+  ...nextConfig,
+  i18n: { locales: ["de"], defaultLocale: "de" },
```
Comment (julialoeschel) [inline next.config.js:20]: 👍

File: pages/singlesite/[slug].js [hunk 3]
```diff
@@ -11,45 +16,234 @@ export default function singleSite() {
...
+        className="mySwiper swiper-h"
+        spaceBetween={0}
+        slidesPerView={1}
+        centeredSlides={true}
+        pagination={{
+          clickable: true,
+        }}
+        modules={[Pagination]}
+        onSlideChange={() => console.log("slide change")}
+        onSwiper={(swiper) => console.log(swiper)}
```
Comment (julialoeschel) [inline pages/singlesite/[slug].js:38]: please don`t leave the logs in your PRs. I guess there should be some functionality. :)

File: pages/singlesite/[slug].js [hunk 4]
```diff
@@ -11,45 +16,234 @@ export default function singleSite() {
...
+              <div className="credits-card">
+                <h2>
+                  {currentSite.creditsCaption + ": "}
+                  {currentSite.coordinates[0] +
+                    " | " +
+                    currentSite.coordinates[1]}
+                </h2>
+                <p className="credits">
+                  {currentSite.credits[0]}
+                  <br />
```
Comment (julialoeschel) [inline pages/singlesite/[slug].js:183]: try not to use </br>. Style it with CSS ;)

File: pages/singlesite/[slug].js [hunk 5]
```diff
@@ -11,45 +16,234 @@ export default function singleSite() {
...
+  overflow: hidden;
+`;
+
 const StyledImage = styled(Image)`
-  object-fit: cover;
-  z-index: -1;
+  display: block;
+  object-fit: contain;
+  width: 80% !important;
+  height: 80% !important;
```
Comment (julialoeschel) [inline pages/singlesite/[slug].js:222]: I know it works. 
keep in mind that !important is not the best choice and shoud be avoided (but it is not always possible, I know)

**Review by julialoeschel — COMMENTED** [review]
great job!
I have some recommendations for you.

keep up the good work !
🐜

File: pages/singlesite/[slug].js (hunk 3 above)
Comment (Elgin-Schultz) [reply pages/singlesite/[slug].js:38]: Will delete them immediately!

File: pages/singlesite/[slug].js (hunk 4 above)
Comment (Elgin-Schultz) [reply pages/singlesite/[slug].js:183]: Will try! (You are talking about the <br>, right?)

File: pages/singlesite/[slug].js (hunk 5 above)
Comment (Elgin-Schultz) [reply pages/singlesite/[slug].js:222]: I know, but unfortunately filll does not work with swiper, so that is the only possibility....

**Comment by Elgin-Schultz** [comment]
Thank you so much for your comments!

File: components/GlobalStyles.js (hunk 1 above)
Comment (Elgin-Schultz) [reply components/GlobalStyles.js:140]: Will have a look...

File: pages/singlesite/[slug].js (hunk 4 above)
Comment (Elgin-Schultz) [reply pages/singlesite/[slug].js:183]: br

**Pull request merged**